server/routes/donationRoutes.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * 
+ * This router handles everything at the /donation endpoint
+ * 
+ */
+
+const express = require('express');
+const donationRouter = express.Router()
+
+const donationController = require('../controllers/donationController.js')
+
+donationRouter.get("/", donationController.getDonations,  (req,res) => {
+  res.status(200).json(res.locals.donations);
+})
```

Comment:
This is correct 100%. However, I suggest that we make the code more readable and easy to follow by explicitly defining the body of the response object like this 
{ 
 donation:  res.locals.donation,
 status: 'fail/success'
}

Reply:
Thanks Khuong,

I've implemented your suggestions in the "add clarity..." commit below.